templates/bacula-dir-header.erb

```diff
@@ -32,5 +32,11 @@ Catalog {
     Name   = MyCatalog
     dbname = "<%= @db_name %>";
     dbuser = "<%= @db_user %>";
-    dbpassword = "<%= @db_pw %>"
+    dbpassword = "<%= @db_pw %>";
+<% if not @db_host.nil? -%>
```

Comment:
You should be able to do `if @db_host` instead of checking if the variable is not nil, which improves readability imo.